App/Sources/DesignSystem/Button/BitgouelButton.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+public struct BitgoeulButton: View {
+    @State public var buttonState: Bool
+    @Environment(\.isEnabled) var isEnabled
+    var text: String
+    
+    public init (
+        text: String = "",
+        buttonState: Bool = false
+    ) {
+        self.text = text
+        self.buttonState = buttonState
+    }
+    
+    private var inputForegroundColor: Color {
+        if isEnabled == false {
+            return Color.bitgouel(.greyscale(.g4))
+        } else if buttonState == true {
+            return Color.bitgouel(.greyscale(.g7))
+        } else {
+            return Color.white
+        }
+    }
+    
+    private var inputBackgroundColor: Color {
+        if isEnabled == false {
+            return Color.bitgouel(.greyscale(.g6))
+        } else if buttonState == true {
+            return Color.bitgouel(.primary(.p4))
+        } else {
+            return Color.bitgouel(.primary(.p5))
+        }
+    }
+    
+    public var body: some View {
+        Button {
+            self.buttonState = true
+        } label: {
+            Text(text)
+        }
+        .bitgouelFont(.text1)
+        .padding(.horizontal, 32)
```

Comment:
```suggestion
        .padding(.horizontal, 32)
```
이렇게 넓이를 정해놓으신 이유가 있을까요?

Reply:
Button 디자인에 왼쪽과 오른쪽에 padding이 32씩 들어가 있어서 그렇게 정했습니다.

Reply:
이렇게 정해놓으면 같은 디자인인데 더 넓이가 긴 버튼이 있을 때 어떻게 하나요??

Reply:
어.... 수정하겠습니다..

Reply:
수정했습니다!!

Reply:
엄 더 넓이가 긴 버튼같은경우에는 그냥 .frame(maxWidth: .infinite) modifier 걸어주면 그냥 길어집니다